app/models/xml/manifest.rb

```diff
@@ -12,7 +14,11 @@ def initialize(translation)
 
       manifest_node = @document.root
       add_manifest_metadata(manifest_node)
-      insert_translated_name(manifest_node)
+
+      phrases = manifest_translated_phrases(manifest_node)
+      # TODO: go strict, what about keeping default title (if not translated)?
```

Comment:
existing specs assumed non-strict translation as the title i18n to exist wasn't 'strongly' required.

Reply:
not sure if this is exactly what you are referring to, but if a `<content:text>` node doesn't contain the `i18n-id` attribute it should be left as is. I'm guessing most of our existing manifests don't have the `i18n-id` attribute for the title, which I will make sure we update to include it everywhere.

Reply:
the assumption here, with existing specs was that given the following : 
```
<?xml version="1.0"?>
<manifest xmlns="https://mobile-content-api.cru.org/xmlns/manifest" xmlns:content="https://mobile-content-api.cru.org/xmlns/content" tool="kgp" locale="en" type="tract">
  <title>
    <content:text i18n-id="89a09d72-114f-4d89-a72c-ca204c796fd9">Knowing God Personally</content:text>
  </title>
</manifest>
```
... the title translation does not need to exist and the package would have been build fine.
if we want to have this 'strict' we simply need to update some package related specs not to assume so

Reply:
ah, I would like to make it strict, if we need to tweak some specs to make it work, then we should

Reply:
believe, it's the a pathological case where the `name_description.xml` would not have a **name** present

Reply:
ah, then this could be dropped. If it's not in OneSky we should let it raise a missing translation error

Reply:
oh wait, I though your last comment was on the other thread